manala.locales/tasks/install.yml

```diff
@@ -9,3 +9,13 @@
     cache_valid_time:   3600
   with_items:
     - locales
+  when: ansible_os_family == "Debian"
+
+- name: RedHat | install > Packages
+  yum:
+    name:  "{{ item }}"
+    state: present
+    update_cache:       true
+  with_items:
+    - glibc-common
```

Comment:
Is ```glibc-common``` the package handling locales on redhat ???

Reply:
it is :) https://fedoraproject.org/wiki/Changes/Glibc_locale_subpackaging or https://unix.stackexchange.com/questions/140299/locale-gen-command-in-centos6